### PR TITLE
Feature/improvments

### LINKED ISSUE #60: extra args to ping people
it would be nice if you could do like `xd docs spigot Player`
and the bot would respond with `@` the given user.
That would make things easier as the person would know that this command executed by me is an answer to his question.

### LINKED ISSUE #65: List Tags does not check args
grafik

### PR DESCRIPTION
**Pull request by DRSchlaubi**
Fixes #65 
Fixes #60

### CODE REVIEW

File: autohelp/kord/build.gradle.kts [hunk 1]
```diff
@@ -38,17 +38,23 @@ repositories {
...
 dependencies {
     api(project(":autohelp"))
-    api("dev.kord", "kord-core", "kotlin-1.5-SNAPSHOT")
+    api("dev.kord", "kord-core", "0.7.3")
 }
 
 tasks {
     withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
         kotlinOptions {
-            jvmTarget = "11"
```
Comment (NyCodeGHG) [inline autohelp/kord/build.gradle.kts:47]: Is there a reson for 15?

Comment (NyCodeGHG) [reply autohelp/kord/build.gradle.kts:47]: nvm there is a reason I cannot read

**Review by NyCodeGHG — APPROVED** [review]
wonderful

File: autohelp/kord/build.gradle.kts (hunk 1 above)
Comment (DRSchlaubi) [reply autohelp/kord/build.gradle.kts:47]: the official reason is that kapt sucks

**Pull request merged**